store/src/java-test/com/zimbra/cs/datasource/imap/RemoteFolderTest.java

```diff
@@ -0,0 +1,136 @@
+package com.zimbra.cs.datasource.imap;
```

Comment:
add the license block, refer to other test classes for same.

Reply:
Done